woocommerce/woocommerce-ios
# Networking: Product details

Mark I for #764.

This PR contains the first Networking call for loading all product details.

To test:
Run the unit tests an ensure they are all green
Did I miss any unit tests?

Update release notes:
